Packs/CommonScripts/Scripts/bitcoinAddressValidator/bitcoinAddressValidator.yml

```diff
@@ -0,0 +1,19 @@
+args:
+- default: false
+  description: The Bitcoin address to verify.
+  isArray: false
```

Comment:
Need to support `isArray: true`.

Reply:
@glicht The script should run once for multiple IOCs?

Reply:
Yes. Take a look at the other format scripts we have for an example.

Reply:
@moishce Please validate the when running the script upon multiple indicators, it is filtering only the real bitcoin urls